test/moclojer/resources/external-body-text.yml

```diff
@@ -0,0 +1,10 @@
+- endpoint:
+    method: GET
+    path: /external-body-text
+    response:
+      status: 200
+      headers:
+        Content-Type: application/json
+      external-body:
+        provider: text
```

Comment:
would be better instead of text be json?

Reply:
it may someone could add .txt as a file without json format and it won´t work right?

Reply:
your comment makes sense, it's an api
hardly anyone will use a form other than json (or xml)